script.js

```diff
@@ -18,21 +25,59 @@ if (data && data.length > 0) {
 }
 
 function AddTodo(todoText) {
-    const element = document.createElement("LI")
-    const elementText = document.createTextNode(todoText)
-    element.appendChild(elementText)
+        const element = document.createElement("LI");
+	const elementBtn = document.createElement("button");
+	const elementText = document.createTextNode(todoText);
+	var elementId = nameId+loop;
+	loop++;
+	elementBtn.id = elementId;
+	element.id = elementId;
+	elementBtn.type = "submit";
+	elementBtn.innerHTML = "x";
```

Comment:
when adding the button like this, please also adjust the styling in the style.css to make sure the app still looks okay.
Here's a screenshot for you:
![image](https://user-images.githubusercontent.com/9539942/79355267-b7b44a00-7f3d-11ea-829e-687600eedd1d.png)

there's a lot of space in between. Please make sure there's not so much space in between.